components/kyma-metrics-collector/pkg/edp/client.go

```diff
@@ -52,6 +54,8 @@ func (eClient Client) NewRequest(dataTenant string) (*http.Request, error) {
 
 	req, err := http.NewRequest(http.MethodPost, edpURL, bytes.NewBuffer([]byte{}))
 	if err != nil {
+		totalRequest.WithLabelValues(fmt.Sprintf("%d", http.StatusBadRequest)).Inc()
```

Comment:
@k15r this shouldn't be a label bad request either? but rather "failed_createRequest" for the same reason as you mentioned for the keb client [here](https://github.com/kyma-project/control-plane/pull/667#discussion_r641636208)